catalogue.py

```diff
@@ -663,8 +664,8 @@ def cutout_box(self, ra, dec, fov=10, redo=False, write=True, verbose=True):
             RA = self.df[self.ra_col]
             DEC = self.df[self.dec_col]
 
-        if (type(ra) is tuple and (type(dec) is not tuple or len(ra) != 2)) or \
-        (type(dec) is tuple and (type(ra) is not tuple or len(dec) != 2)):
+        if (type(ra) is tuple and (type(dec) is not tuple or len(ra) != 2)) or\
+         (type(dec) is tuple and (type(ra) is not tuple or len(dec) != 2)):
```

Comment:
indent this line with 8 spaces compared to the previous line (flake8)

Reply:
Done.